moped-editor/src/queries/signals.js

```diff
@@ -1,28 +1,24 @@
 import { gql } from "@apollo/client";
 
-// Status ID 6 is archived (soft deleted)
 // Status ID 3 is cancelled
 
 export const SIGNAL_PROJECTS_QUERY = gql`
   query SignalProjectsQuery {
     moped_project(
       where: {
+        is_retired: { _neq: true }
         _or: [
           {
             moped_proj_components: {
               moped_components: { component_name: { _ilike: "signal" } }
+              status_id: {_eq: 1}
```

Comment:
This is totally fine, but like we were discussing, we may want to consider a look-up table to get these magic numbers out.

Reply:
agreed. Should we go ahead and make a tech debt issue for this? cc/ @amenity @johnclary